demos/share-files.html

```diff
@@ -118,10 +118,20 @@ <h1>
         textfield.value = '';
     }
 
+    function checkBasicFileShare() {
+      // XXX: There is no straightforward API to do this.
```

Comment:
I think this is a problem, as already mentioned in https://github.com/w3c/web-share/issues/108#issuecomment-697689527. Something like this would help.

```js
navigator.canShare("files"); // basic feature support
navigator.canShare("files", [files]); // matches canShare({files});
```

Reply:
https://github.com/w3c/web-share/pull/184 was about `canShareFiles` with file types, but since Blink is now also checking file sizes, I guess receiving a sequence of files is needed?

Reply:
canShare in Blink seemingly has no check for file types nor file sizes, and share() somehow throws PermissionError instead. Maybe to prevent fingerprinting? But then what's the point of canShare with the current form? 👀

Reply:
hehe, "incremental improvement"?